pkg/api/controller.go

```diff
@@ -2996,6 +3014,36 @@ func paginationAmount(v *PaginationAmount) int {
 	return i
 }
 
+func resolvePathList(objectList *[]string, prefixList *[]string) []catalog.PathRecord {
+	var pathList []catalog.PathRecord
```

Comment:
a suggestion, it's up to you
```suggestion
	var pathRecords []catalog.PathRecord
```